packages/turbo-types/schemas/schema.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
I would prefer not checking in codegen artifacts if it can be avoided. Not blocking, but if it can be avoided we should probably rely on `turbo` caching instead of `git`.

Reply:
We can do that, but there are some downsides I want to get your thoughts on:

How will we know _how_ it changes the schema when a type is updated?  Like, in a code review how can I see changes to the schema happening (or not happening) on a given PR?

The other reason to do it is that this file is used in the repo (by vscode) and we need something to point to, but if you haven't built yet there will never be anything there.

Reply:
What's your plan for keeping this up to date and making sure the schema actually gets build if there are changes here?

Also are you thinking about making this artifact the source of truth?